packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -74,44 +75,54 @@ export default class EmailCardDropRequestsRoute {
       return;
     }
 
+    ctx.type = 'application/vnd.api+json';
+
+    let claimedWithUserAddress = await this.emailCardDropRequestQueries.query({
+      ownerAddress: ctx.state.userAddress,
+      claimedAt: NOT_NULL,
+    });
+
+    if (claimedWithUserAddress.length > 0) {
+      ctx.status = 422;
+      ctx.body = {
+        errors: [
+          {
+            status: '422',
+            title: 'Address has already claimed a prepaid card',
+          },
+        ],
+      };
+
+      return;
+    }
+
     let email = ctx.request.body.data.attributes.email;
 
     let hash = crypto.createHash('sha256');
     hash.update(email);
     let emailHash = hash.digest('hex');
 
-    let verificationCode = (crypto.randomInt(1000000) + '').padStart(6, '0');
+    let claimedWithUserEmail = await this.emailCardDropRequestQueries.query({
+      emailHash,
```

Comment:
This isn't reliable deduplication because of the `johnsmith+hello@gmail.com` thing.

Reply:
Maybe we need to change our hashing strategy.

Reply:
Or simply reject email addresses with +, to make things easier.

Reply:
WELL the `validator` package actually [includes](https://github.com/validatorjs/validator.js#sanitizers) a `normalizeEmail` to strip those out, thoughts???

Reply:
Seems like we could try that out!